clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/UbamTests.scala

```diff
@@ -331,7 +331,7 @@ trait UbamTests { self: BaseIntegrationSpec =>
       }
       rpIdResults should have length 1
       rpIdResults.headOption.flatMap(_.researchProjectId) should be(
-        researchProjectIds.lastOption
+        Some(researchProjectIds.lastOption)
```

Comment:
I don't think you want the `Some` wrapper here... you'll end up with `Option[Option[_]]`